app/jobs/scheduled/encrypted_post_timer_evaluator.rb

```diff
@@ -20,11 +21,11 @@ def execute(args)
     end
 
     def posts_to_delete(encrypted_post_timer)
-      @post ||= Post.with_deleted.find_by(id: encrypted_post_timer.post_id)
-      return [] unless @post
-      @topic ||= Topic.with_deleted.find_by(id: @post.topic_id)
-      @posts_to_delete ||= @post&.is_first_post? ? @topic.posts.with_deleted.order(created_at: :desc) : [@post]
-      @posts_to_delete.compact
+      post = Post.with_deleted.find_by(id: encrypted_post_timer.post_id)
+      return [] unless post
```

Comment:
Sam will hunt you down for `unless` usage ;) `return if !post` or `return if post.blank?` instead